### DISCUSSION TITLE
Mirorred colors/ wrong colors placement on the strip

### DISCUSSION THREAD

**Discussion by emanueletocci** [hyperion-project/hyperion.ng#1655] [Q&A]
Hi, I'm using Hyperion with an arduino nano and ws2812b as cheap ambilight... It works but the placement of colors is wrong... it is mirrored.
How can i fix it?

![photo_2023-11-11_15-23-12](https://github.com/hyperion-project/hyperion.ng/assets/83188429/c1846abe-c434-4aff-936e-8b23c78df9e8)

My config file (there aren't any personal data inside, I removed network data):

[Hyperion-2.0.15-Backup (My Hyperion Config) 2023.11.11.json](https://github.com/hyperion-project/hyperion.ng/files/13325856/Hyperion-2.0.15-Backup.My.Hyperion.Config.2023.11.11.json)

Hyperion version: 2.0.15 stable
Arduino sketch: https://github.com/hyperion-project/hyperion.ng/blob/master/assets/firmware/arduino/adalight/adalight.ino
OS: Windows 11 

Thank you everybody.

**Comment by Lord-Grey** [comment]
You need to go to the layout configuration and change the layout in line with your LED strip direction.
